server/documents/modules/calendar.html.eco

```diff
@@ -222,6 +234,15 @@ themes      : ['Default']
       ;
       </div>
     </div>
+    <div class="another example">
+      <p>You can also set a minimum and a maximum selection date using HTML markup</p>
+      <div class="ui calendar" id="minmax_calendar_2" data-min-date="10/15/2018 08:00:00" data-max-date="10/19/2018 18:00:00">
```

Comment:
There should be some information regarding Date-Format (It's MM/DD/YYYY if MM <=12, otherwise it's DD/MM/YYYY):
10/15/2018 is October, 15th 2018 (MM/DD/YYYY), 
15/10/2018 is also detected as October, 15th 2018, but
05/04/2018 is always May, 4th 2018